### DISCUSSION TITLE
should we enable mergify on the samba-build repository?

### DISCUSSION THREAD

**Discussion by obnoxxx** [samba-in-kubernetes/samba-build#17] [Polls]
IMHO, mergify has proven very useful on  other  sink repos for automating and streamlining the review and merge process.  this is a poll to get a sense of whether we should Should set it up for this one as well.

**Comment by obnoxxx** [comment]
@gd , @anoopcs9 , @synarete , @spuiuk , @gd  please share your opinion by voting ... :-)

**Comment by spuiuk** [comment]
No strong feelings either way.. but once setup, very little maintenance required. so why not?

**Comment by obnoxxx** [comment]
@xhernandez, we'd like your opinion on this, too
